nexus/db-queries/src/db/datastore/vpc.rs

```diff
@@ -1684,9 +1648,9 @@ mod tests {
             .await
             .expect("failed to insert blueprint");
 
-        // We haven't set a blueprint target yet, so we should still only see
-        // the two RSS-inserted service-running sleds.
-        assert_eq!(&harness.sled_ids[..2], fetch_service_sled_ids().await);
+        // We haven't set a blueprint target yet, so we should still fail to see
+        // any services on sleds.
+        assert_eq!(&[] as &[Uuid], fetch_service_sled_ids().await);
```

Comment:
Could you turn this into an `assert_service_sled_ids(expected: &[Uuid])`?

Reply:
Done in 7de458e49; that is much nicer - thanks!